app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -24,6 +24,46 @@ interface Props {
   onSubmit: () => void;
 }
 
+// passing content suffix to fields that need nominator and denominator (example: tCO2e/GJ)
+const createEmissionIntensityReportUiSchema = (
```

Comment:
We can just take in `newFormData` here and grab what we want off of it. That will make the call to it in `app/components/Form/ProjectEmissionIntensityReportForm.tsx` a little cleaner.